### PR TITLE
Audit Sentinel Detection Rules

### PR DESCRIPTION
**Pull request by samikroy**
Required items, please complete
   
   Change(s):
   - Added a query to audit Sentinel Detection rules changes,
   - Added a saved query to be deployed as function.

   Reason for Change(s):
   - New tool addition.

   Testing Completed:
   - Tested with pre-requisites qualified and works good.

### CODE REVIEW

**Comment by v-atulyadav** [comment]
Hi @v-sabiraj @v-rucdu,
Please provide your feedback on this.
Thanks

**Comment by samikroy** [comment]
Hi @v-sabiraj  @v-rucdu & @v-atulyadav 
Please provide your inputs on this.
Thanks

**Comment by samikroy** [comment]
Hi @v-sabiraj @v-rucdu & @v-atulyadav
Please provide your inputs on this.
Thanks

**Comment by v-atulyadav** [comment]
Looking into this

**Comment by v-atulyadav** [comment]
Hi @v-sabiraj @v-rucdu,
Please provide your feedback on this.
Thanks

**Comment by v-prasadboke** [comment]
Hello @v-rucdu and @v-sabiraj waiting for your feddback

File: Tools/Audit-Sentinel-Detection/Readme.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+
+
+# Audit Sentinel Detection Rules
+
+### Prerequistes : [Configure Audit in Microsoft Sentinel](https://learn.microsoft.com/en-us/azure/sentinel/enable-monitoring)
```
Comment (v-sabiraj) [inline Tools/Audit-Sentinel-Detection/Readme.md:5]: Hey @samikroy, please fix the typo Prerequistes to Prerequisites

File: Tools/Audit-Sentinel-Detection/Readme.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+### Purpose : Detection / Analytics are dircetly changed in the production instace at times due to several reaons.
```
Comment (v-sabiraj) [inline Tools/Audit-Sentinel-Detection/Readme.md:7]: @samikroy, please fix the below typos 

dircetly to directly 
instace to instance
 reaons to reasons

File: Tools/Audit-Sentinel-Detection/Readme.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+ + As the DevOps workflow is not set up.
+ + Due to urgency.
+
+Here is KQL query to idetify such scenarios and an azure function developped in KQL which can be used to display the changes.
```
Comment (v-sabiraj) [inline Tools/Audit-Sentinel-Detection/Readme.md:12]: @samikroy , fix these typos as well idetify to identify,  developped to developed

Comment (samikroy) [reply Tools/Audit-Sentinel-Detection/Readme.md]: Fixed

File: Tools/Audit-Sentinel-Detection/Readme.md (hunk 2 above)
Comment (samikroy) [reply Tools/Audit-Sentinel-Detection/Readme.md]: Fixed

File: Tools/Audit-Sentinel-Detection/Readme.md (hunk 1 above)
Comment (samikroy) [reply Tools/Audit-Sentinel-Detection/Readme.md]: Fixed

**Comment by v-atulyadav** [comment]
Hi @sreedharande, please provide your feedback on this. Thanks

**Comment by v-prasadboke** [comment]
Hello @sreedharande waiting for your feedback

**Comment by v-prasadboke** [comment]
Hello @sreedharande waiting for your feedback

**Comment by samikroy** [comment]
Hi All,
Could you please help with the merge of this 
or let me know for any updates to it.
Thank you.

**Comment by samikroy** [comment]
Hi @sreedharande , @v-prasadboke , @v-atulyadav , @v-sabiraj ,
Request your help for the merger of this PR.
Thank you,

**Comment by v-atulyadav** [comment]
Hi @samikroy, could you please add support information to the readme file? Thanks

**Comment by v-atulyadav** [comment]
Hi @samikroy, adding the following details to your read me file will help users to get support if they have any issues.

{
	"support": {
	  "name": "Publisher Name/ID",
	  "email": "Email for template Support",
	  "link": "Link of Support contacts for this template"
	}
}

**Comment by samikroy** [comment]
> Hi @samikroy, adding the following details to your read me file will help users to get support if they have any issues.
> 
> { "support": { "name": "Publisher Name/ID", "email": "Email for template Support", "link": "Link of Support contacts for this template" } }

@v-atulyadav  - Please have a look and let me know.

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**